docs/docs/populate-connections-views.md

```diff
@@ -34,6 +34,14 @@ server.workbooks.populate_connections(workbook)
 print([connection.datasource_name for connection in workbook.connections])
 ```
 
+## Update connections for workbooks
+
+```py
+server.workbooks.populate_connections(workbook)
+workbook.connections[i].server_address = 'new_endpoint'
```

Comment:
This is technically correct, but I don't like the pattern (Not your fault, the way we try to keep the models and the upstream in sync are a little wonky)

Can you document it like this:
```py
server.workbooks.populate_connections(workbook)
conn_to_update = workbook.connections[0]  # Or whichever you want, but i is context-less and can't be copy/pasted
conn_to_update.server_address = 'abcdefg'
conn_to_update.port = 1337
server.workbooks.update_conn(workbook, conn_to_update)
```

/cc @RussTheAerialist